src/components/elements/generatedManuscriptText.tsx

```diff
@@ -0,0 +1,208 @@
+        }
+
+        if (line_range_per_column != null) {
+            lines = <> and approximately <b>{line_range_per_column}</b> lines per column</>
+        }
+
+        if (folio_start_of_the_tm_part != null) {
+            folio_start = <> The Marian miracle stories begin on folio <b>{folio_start_of_the_tm_part}</b> of the whole manuscript.</>
+        }
+
+        return <>This manuscript has a total of {folios}{pages}{scans}. It has {columns} {lines}. {folio_start}</>
+    }
+
+    // duplicate/missing_scans/rebound_in_disorder= COLUMN DOESNT EXIST IN SHEET --> USE UNDERSCORES AND NOT "/"
+    const DeterminePageProblemSentence = (disordered_rebound: string, duplicate_or_missing: string) => {
+        if (disordered_rebound != null && duplicate_or_missing != null) {
+            return <> Unfortunately, this manuscript was bound in disorder and digitized improperly so that some folios were skipped or duplicated.</>
+        }
+        else if (disordered_rebound != null) {
+            return <> Unfortunately, this manuscript was bound in disorder.</>
+        }
+        else if (duplicate_or_missing != null) {
+            return <> Unfortunately, this manuscript was digitized improperly and some folios were skipped or duplicated.</>
+        }
+        return
+    }
+
+    // SEE TICKET ABOUT HYPERLINK TO STAFF PAGE
+    const DetermineCatalogerSentence = (pemm_cataloger_name: string, print_cataloger_name: string, pemm_volunteer_name: string) => {
+        let s1 = null
+        let s2 = null
+        if (pemm_cataloger_name != null && print_cataloger_name != null) {
+            s1 = <> The stories in this manuscript were cataloged by <b>{pemm_cataloger_name}</b> and <b>{print_cataloger_name}</b>.</>
+        }
+        else {
+            if (pemm_cataloger_name != null) {
+                s1 = <> The stories in this manuscript were cataloged by <b>{pemm_cataloger_name}</b></>
+            }
+            else if (print_cataloger_name != null) {
+                s1 = <> The stories in this manuscript were cataloged by <b>{print_cataloger_name}</b></>
+            }
+        }
+
+        if (pemm_volunteer_name != null) {
+            s2 = <> <b>{pemm_volunteer_name}</b> provided assistance by typing incipits.</>
+        }
+
+        return <> {s1} {s2}</>
+    }
+
+    // parts n, m, p instructions are missing from issue post
+    // WHERE TO GET UPLOAD DATE + TOTAL NUM OF MANUSCRIPTS
+    return (
+        <>
+            <div>
+                <h2>
+                    This <b>{manuscript.language}</b> language manuscript was created between <b>{manuscript.date_range_start}</b> and <b>{manuscript.date_range_end}</b>.
+                    {DetermineManuscriptDateBasis(manuscript.date_note)}
+                </h2>
+                <h2>
+                    This manuscript was cataloged, digitized, or purchased in (and thus may have been created in or near) [CANT FIND PROVENANCE COLUMN] in [OR PLACE_RECORDED COLUMN].
```

Comment:
[CANT FIND PROVENANCE COLUMN] in [OR PLACE_RECORDED COLUMN] this doesn't seem right